pandas/tests/io/parser/test_common.py

```diff
@@ -1151,6 +1152,7 @@ def test_chunks_have_consistent_numerical_type(all_parsers):
     assert result.a.dtype == float
 
 
+@pytest.mark.skip("unreliable test #35214")
```

Comment:
1) should this be xfail?
2) i think the solution will end up involving using td.check_file_leaks in a lot of places.  Is there a non-verbose way to run it once at the module level?

Reply:
1. Yes, it should be xfail. I'll submit a PR to fix this.
2. Haven't thought about this. Might need some discussion.